src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -9202,18 +9218,25 @@ void CEEInfo::getFunctionFixedEntryPoint(CORINFO_METHOD_HANDLE   ftn,
 
     pResult->accessType = IAT_VALUE;
 
-
-#ifndef CROSSGEN_COMPILE
-    // If LDFTN target has [NativeCallable] attribute , then create a UMEntryThunk.
+// Also see GetBaseCompileFlags() below for an additional check.
+#if defined(TARGET_X86) && defined(TARGET_WINDOWS) && !defined(CROSSGEN_COMPILE)
+    // Deferring X86 support until a need is observed or
+    // time permits investigation into all the potential issues.
     if (pMD->HasNativeCallableAttribute())
     {
         pResult->addr = (void*)COMDelegate::ConvertToCallback(pMD);
```

Comment:
Can we make `COMDelegate::ConvertToCallback` ifdefed out with the same condition?

Reply:
As in removing the function entirely? The linker would normally remove a function if it isn't used.

Reply:
It is more just for clarify that the function is used for Windows x86 only.